### ISSUE TITLE
Query values from the db? And Updating db version?

### ISSUE THREAD

**Issue by ekahannah** [skiptools/skipapp-databake#1]
Hi, 
I noticed that this app only showcased creating of db as well as insert values and delete of db.  I was wondering if we also have an example of querying statement ? For example `Select * FROM DBTABLE` or `Select STRING FROM DBTABLE`

I tried doing the ff: 
```            
let stmnt = try database.query(sql: "SELECT STRING FROM \(tableName)")
            for value in stmnt {
                print("Query \(String(describing:   value.first))")
            }
```
            
But the value I get is :
`Optional(Optional(""))` 
I cant seem to get the UUIDString that was inserted.

Also, Does this also support updating of db version when certain attributes are added in the database?

**Comment by marcprux** [comment]
Not sure why that would be failing. I've just pushed some enhancements to the DataBakeModel module that has parameterized selects and updates: https://github.com/skiptools/skipapp-databake/blob/main/Sources/DataBakeModel/DataBakeModel.swift#L256

Can you try running the tests for `DataBakeModel` and let us know if they pass (be sure to `Update to Latest Package Versions` to get the most recent SkipSQL as well)? The sample is still very much a work in progress, and doesn't support editing individual records yet, but you can see examples of creating and deleting records and support for large result sets.

We'd love to hear your feedback and suggestions for enhancement!

<img width="1920" alt="Screenshot 2023-11-20 at 07 06 55" src="https://github.com/skiptools/skipapp-databake/assets/659086/399ab80c-479a-43c2-8f38-673b48db4572">

**Comment by ekahannah** [comment]
Hi @marcprux , 
Unfortunately, I can't run the latest databake app. Here is the error when opening on xcode.
![Screenshot 2023-11-21 at 9 39 55 AM](https://github.com/skiptools/skipapp-databake/assets/1568192/93850f0b-b49d-46fc-a3b3-545ea6121f4c)

I was able to run 0.0.2 tag though

**Comment by hannah-smudge** [comment]
Hi @marcprux ,

I successfully run the app after reimporting the project. 
However, I noticed that the title, and contents are not being saved on the db. Only the id and created values are saved
Please see the table below.
![Screenshot 2023-11-24 at 1 58 54 PM](https://github.com/skiptools/skipapp-databake/assets/97433938/f32e5a42-3460-4da2-92bd-ac020e72060d)

Do you know the reason why?

**Comment by ekahannah** [comment]
Hi @marcprux ,

I successfully run the app after reimporting the project.
However, I noticed that the title, and contents are not being saved on the db. Only the id and created values are saved
Please see the table below.
I think there may be an issue with inserting texts into db
![Screenshot 2023-11-24 at 1 58 54 PM](https://github.com/skiptools/skipapp-databake/assets/1568192/d248d747-da3c-43b5-b593-bf8e51cee0aa)

**Comment by marcprux** [comment]
This was a bug in `skip-sql` – bound string (and blob) parameters were not being handled correctly. It's been fixed in 0.3.8. If you `Update to Latest Package Versions` from the Xcode `File`/`Packages` menu and rebuild, it should work.

Thanks for reporting the issue.
